docs/reference/developer/getting-started.rst

```diff
@@ -1,22 +1,103 @@
 Getting Started
 ===============
 
-Prerequisites
-~~~~~~~~~~~~~
+TL;DR
+-----
 
-All non-optional build dependencies are installed automatically through Conan when running CMake.
-Conan itself can be installed manually but we recommend using the ``developer.yml`` file
-for installing this and other dependencies in a ``conda`` environment (see below).
-Alternatively you can refer to this file for a full list of dependencies.
+Standard setup
+~~~~~~~~~~~~~~
 
+.. code-block:: bash
+
+  # Get the code
+  git clone git@github.com:scipp/scipp.git && cd scipp
+  git submodule init
+  git submodule update
+
+  # Setup dev env
+  mamba env create -f docs/environments/developer.yml
+  conda activate scipp-dev
+  pre-commit install
+  pre-commit run --all-files
+
+  # Build, install, and run C++ tests
+  cmake --preset base
+  cmake --build --preset build
+  ctest --preset test
+
+  # Setup editable install and run Python tests
+  tox -e editable
+  conda develop src
+  python -m pytest tests
+
+You should now also be able to run ``python -c 'import scipp as sc'``.
+
+Common tasks
+~~~~~~~~~~~~
+
+All of the below assume that the standard setup outlined above was performed.
+
+Build the docs:
+
+.. code-block:: bash
+
+   tox -e docs
+
+Update the pip requirements via ``pip-compile-multi``:
+
+.. code-block:: bash
+
+   tox -e deps
+
+Prepare a release (make sure to give the desired release instead):
+
+.. code-block:: bash
+
+   tox -e prepare-release -- 23.01.0
+
+Overview
+--------
+
+We provide a Conda environment file with developer dependencies in our Git repository.
```

Comment:
So requirements are conda/mamba and conda-build (used for `conda develop`)

Reply:
`conda-build` is in the env file.

Reply:
Oh yeah :)